tests/lib/test_organizations.py

```diff
@@ -0,0 +1,49 @@
+from pathlib import Path
+
+from lib.organization import (
+    contains_one_organization_per_file,
+    get_organizations,
+    get_organizations_path,
+)
+
+
+def test_get_org_paths() -> None:
+    org_paths = get_organizations_path(
+        Path("tests/fixtures/with_well_formatted_organizations")
+    )
+    assert len(org_paths) == 2
+    org_1 = org_paths[0]
+
+    expectedPath1 = (
```

Comment:
(Nit) En Python on utilise par convention le `snake_case`

```suggestion
    expected_path_1 = (
```